app/Dockerfile.base

```diff
@@ -3,7 +3,7 @@ FROM quay.io/azavea/django:1.11-python2.7-slim
 # N.B. This will not work for docker versions above 1.10.x; Docker 1.11 and above contain
```

Comment:
Can this comment be deleted now?

Reply:
It appears so - in theory something must have happened previously to make it outdated because I didn't notice any problems for this, but now that we are on >1.10.x it's pretty clearly obsolete now.